tox.ini

```diff
@@ -7,7 +7,7 @@ envlist = py
 
 [testenv]
 description = Lints code and runs all agent and server unit/functional tests
-install_command = pip install --no-cache-dir --progress-bar off --prefix={envdir} {opts} {packages}
+install_command = pip install --cache-dir={toxworkdir}/cache --progress-bar off --prefix={envdir} {opts} {packages}
```

Comment:
Hmm; so we're using two separate cache directories: `/var/tmp/USER/cache` and `/var/tmp/USER/tox/cache`. Does this matter? I think I've lost track of the contexts. At worst, we're duplicating some cache and it's still probably better than before (and pragmatically no different for Jenkins).

Reply:
The cache directory use in `jenkins/python-setup.sh` is really of no use because it is for an ephemeral drive.  So best to just remove it.

Reply:
How much trouble would we get into if we mapped the cache directory from a persistent volume?  (How good is pip's cache management -- would it be able to determine when an entry is stale or bad and discard it?)

Reply:
It is supposed to be able to do that.  The problem is where to put the volume.  I'd rather approach that in a separate effort since it will involve environmental factors.  We might want to look at a cache that works for all Jenkins instances as well.